.go-version

```diff
@@ -1 +1 @@
-1.18.5
+1.19
```

Comment:
```suggestion
1.19.6
```

Since this is also used in the build workflow, we should use the latest to get [the CVE fixes](https://github.com/golang/go/issues?q=milestone%3AGo1.19.6+label%3ACherryPickApproved+).